hi-ml-histopathology/src/histopathology/preprocessing/loading.py

```diff
@@ -99,12 +99,12 @@ def __call__(self, data: Dict) -> Dict:
 
         # cuCIM/OpenSlide takes absolute location coordinates in the level 0 reference frame,
         # but relative region size in pixels at the chosen level
-        origin = (level0_bbox.x, level0_bbox.y)
+        origin = (level0_bbox.y, level0_bbox.x)
         scale = image_obj.resolutions['level_downsamples'][self.level]
         scaled_bbox = level0_bbox / scale
 
         data[self.image_key], _ = self.reader.get_data(image_obj, location=origin, level=self.level,
-                                                       size=(scaled_bbox.w, scaled_bbox.h))
+                                                       size=(scaled_bbox.h, scaled_bbox.w))
```

Comment:
Now I understand why the axis were swapped before - why the need to swap axis in this transform? 

I couldn't find a test for this transform, this could be something worth to add as the transform was used in more than one place (e.g.  for the creation of TCGA-PRAD https://github.com/microsoft/hi-ml/blob/main/hi-ml-histopathology/src/histopathology/preprocessing/create_tiles_dataset.py)

Reply:
Not sure if anything changes in https://github.com/microsoft/hi-ml/blob/main/hi-ml-histopathology/src/histopathology/preprocessing/create_tiles_dataset.py ? This was used for tiling other slide datasets

Reply:
> Now I understand why the axis were swapped before - why the need to swap axis in this transform?

Because the reader expects (y, x) coordinates.

> I couldn't find a test for this transform, this could be something worth to add as the transform was used in more than one place (e.g. for the creation of TCGA-PRAD https://github.com/microsoft/hi-ml/blob/main/hi-ml-histopathology/src/histopathology/preprocessing/create_tiles_dataset.py)

As I said above, I would wait to see if we need to create more tiles datasets before we add tests. Maybe we can add an issue to the backlog.

Reply:
@harshita-s I'm not sure what you mean; I haven't touched that file.

Reply:
Some more context about the coordinate swapping:
- https://github.com/microsoft/hi-ml/issues/310

Reply:
> @harshita-s I'm not sure what you mean; I haven't touched that file.

Was just wondering if we need to change that file as well (Daniel and Valentina suggested that it should be changed but this will be addressed in a later PR)

Reply:
Ok, thanks for clarifying!

Reply:
@fepegar, I think @harshita-s is wondering whether your changes in this PR affect the other script (e.g. via `LoadROId`, as @vale-salvatelli mentioned). We may need to revisit `create_tiles_dataset.py` somewhat soon e.g. for TCGA-PRAD. I'd be in favour of merging this now and discussing separately how to harmonise the scripts going forward.

Reply:
Sounds good!

Reply:
I am OK with getting in delaying this test if @dccastro is going to refactor and look into TCGA-PRAD soon (before we forget about this change). Adding a warning or a TODO in `create_tiles_dataset` would not harm